### DISCUSSION TITLE
How to Locally + Topic modelling (Non-tech user)

### DISCUSSION THREAD

**Discussion by RazyRo** [brianpetro/obsidian-smart-connections#659] [Q&A]
I love how the app connects points within my notes! With features like Notes Embedding and Blocks Embedding, I was hoping for an easy way to link multiple notes/blocks.

I found your tutorial, 'Getting Started with Smart Connections and Smart Connect Software,' and it includes local chat models like Llama 3 through Ollama and LM Studio on GitHub. Could you make a tutorial similar to local LLM for non-technical Obsidian users like us?

Also, could you tell us how the smart connect feature ensures it understands the context and meaning of the notes, rather than just focusing on individual words? Does it use any advanced techniques to find connections between notes beyond just word similarity? Also, does it use any methods to group similar notes or ideas together to identify common themes?

a few search results as I don't know the exact solution:
https://arxiv.org/abs/2402.11398
https://arxiv.org/html/2403.16248v1

![Screenshot (6xj00)](https://github.com/brianpetro/obsidian-smart-connections/assets/143066812/5dc0595a-d6e2-4bc1-8b17-3753cc682200)

**Comment by hittjwX** [comment]
That's a great question I'm following. Most of my work is technical writing and articles. Everything is coded with campaign and source codes for marketing.

I'm unsure if AI has figured out that these codes in the front matter, file paths, and hashtags are part of a taxonomy.

Would I need to have a definition document so the model could learn? Or would it recognize that certain documents tend to link each other with similar coding and topics?
